tests/UI/pages/BO/catalog/products/add.js

```diff
@@ -407,6 +414,34 @@ class AddProduct extends BOBasePage {
   async goToCatalogPage(page) {
     await this.clickAndWaitForNavigation(page, this.goToCatalogButton);
   }
+
+  /**
+   * Add product to pack
+   * @param page
+   * @param product
+   * @param quantity
+   * @returns {Promise<void>}
+   */
+  async addProductToPack(page, product, quantity) {
+    await page.type(this.packItemsInput, product);
+    await this.waitForSelectorAndClick(page, this.packsearchResult);
+    await this.setValue(page, this.packQuantityInput, quantity.toString());
+    await page.click(this.addProductToPackButton);
+  }
+
+  /**
+   * Add pack of products
+   * @param page
+   * @param pack
+   * @returns {Promise<void>}
+   */
+  async addPackOfProducts(page, pack) {
+    const keys = Object.keys(pack);
+    /*eslint-disable*/
```

Comment:
It's better to use `eslint-disable-next-line` otherwise it disables eslint for all lines after this comment.

Reply:
or use something like this to not disable eslint
```js
    for (let i = 0; i < keys.length; i+=1) {
      await this.addProductToPack(page, keys[i], pack[keys[i]]);
    }
```

Reply:
Done thx :+1: